example-context/example-service/example-event/example-event-processor/src/raml/cakeshop-event-processor.messaging.raml

```diff
@@ -0,0 +1,14 @@
+#%RAML 0.8
+title: API Using media type in the URL
+baseUri: message://event/processor/message/cakeshop
+version: v1
+/cakeshop.event:
+  post:
+    body:
+        application/vnd.cakeshop.recipe-added+json:
+            schema: !include json/schema/cakeshop.recipe-added.json
+            example: !include json/add-recipe.json
+
+        application/vnd.cakeshop.cake-made+json:
+            schema: !include json/schema/cakeshop.cake-made.json
+            example: !include json/make-cake.json
```

Comment:
cake-made, made-cake - should we have consistent names? - not sure if this is systemic